docs/resources/cpts_task.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "Cloud Performance Test Service (CPTS)"
+---
+
+# huaweicloud_cpts_task
+
+Manages a pressure Test Task resource within HuaweiCloud CPTS.
+The task resource only supports serial execution mode.
+
+## Example Usage
+
+```hcl
+resource "huaweicloud_cpts_project" "test" {
+  name = "tf_demo_project"
+}
+
+resource "huaweicloud_cpts_task" "test" {
+  name       = "tf_demo_task"
+  project_id = huaweicloud_cpts_project.test.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the task resource. If omitted, the
+  provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of task, which can contain a maximum of 42 characters.
+
+* `project_id` - (Required, Int, ForceNew) Specifies the CPTS project ID which the task belongs to.
+ Changing this parameter will create a new resource.
+
+* `benchmark_concurrency` - (Optional, Int) Specifies benchmark concurrency of the task, the value range is 0 to 2000000.
+  Reference for the calculation of the number of concurrent users.
+  `Number of concurrent users` = `benchmark concurrency` * `concurrency ratio`. The default value is `100`.
```

Comment:
suggest to move `the default value is 100` to `value range` above. this makes me feel like the number of concurrent users's default value is 100

Reply:
ok